docker/run_cypress_tests/Dockerfile

```diff
@@ -44,20 +46,12 @@ RUN which yarn; yarn --version
 # yarn avaialble instead of our specified version
 RUN yarn set version "${YARN_VERSION}"
 
-# Setup Postgres repository
-# see https://wiki.postgresql.org/wiki/Apt
-RUN apt-get update
-RUN apt-get install -y lsb-release
-RUN curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null
-RUN sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
-# update again with the new repository for postgres added
-RUN apt-get update
-RUN apt-get install -y postgresql-client-13
-
 # Install other dependencies
+# Note that postgresql-client-13 is available through debian bullseye
+# that the ruby image is based on
 RUN apt-get install -y --no-install-recommends \
```

Comment:
```suggestion
RUN apt update && \
    apt-get install -y --no-install-recommends \
```
You deleted all the `apt updates` above, one is needed tho:

```
 => ERROR [mampf stage-1 11/20] RUN apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget wai  0.8s
------
 > [mampf stage-1 11/20] RUN apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget wait-for-it:
0.571 Reading package lists...
0.589 Building dependency tree...
0.591 Reading state information...
0.594 Package ghostscript is not available, but is referred to by another package.
0.594 This may mean that the package is missing, has been obsoleted, or
0.594 is only available from another source
0.594
0.594 Package ffmpeg is not available, but is referred to by another package.
0.594 This may mean that the package is missing, has been obsoleted, or
0.594 is only available from another source
0.594
0.595 E: Package 'ffmpeg' has no installation candidate
0.595 E: Unable to locate package pdftk
0.595 E: Package 'ghostscript' has no installation candidate
0.595 E: Unable to locate package libarchive-tools
0.595 E: Unable to locate package postgresql-client-13
0.595 E: Unable to locate package sqlite3
0.595 E: Unable to locate package wait-for-it
------
failed to solve: process "/bin/bash --login -c apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget wait-for-it" did not complete successfully: exit code: 100
```